mmhuman3d/models/losses/mse_loss.py

```diff
@@ -158,3 +158,41 @@ def forward(self,
             sigma=self.sigma)
 
         return loss
+
+
+@LOSSES.register_module()
+class PoseRegLoss(nn.Module):
+    """Regulizer loss for body pose parameters.
+
+    Args:
+        reduction (str, optional): The method that reduces the loss to a
+            scalar. Options are "none", "mean" and "sum".
+        loss_weight (float, optional): The weight of the loss. Defaults to 1.0
+    """
+
+    def __init__(self, reduction='mean', loss_weight=1.0):
+        super().__init__()
+        assert reduction in (None, 'none', 'mean', 'sum')
+        self.reduction = reduction
+        self.loss_weight = loss_weight
+
+    def forward(self,
+                body_pose,
+                weight=None,
+                avg_factor=None,
+                loss_weight_override=None,
+                reduction_override=None):
+        reduction = (
+            reduction_override if reduction_override else self.reduction)
+        loss_weight = (
+            loss_weight_override
+            if loss_weight_override is not None else self.loss_weight)
+
+        pose_prior_loss = loss_weight * (body_pose**2)
```

Comment:
If PoseRegLoss works by avoiding extreme joint angles, it would be better considered as a prior loss, and place in prior_loss.py.

Reply:
Agreed. PoseRegLoss has been moved to prior_loss.py in https://github.com/open-mmlab/mmhuman3d/pull/102/commits/f8277523a4abbf4c4c01a592c3a19be73f93ce27